<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtStringDialog</class>
 <widget class="QDialog" name="QtStringDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>535</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>QtStringDialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label1">
       <property name="text">
        <string>输入</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editX"/>
     </item>
     <item>
      <widget class="QPushButton" name="m_button">
       <property name="text">
        <string>排序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label2">
       <property name="text">
        <string>倒序</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="m_editY"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
